components/CircleChart.js

```diff
@@ -1,10 +1,55 @@
-import React, { useContext } from 'react';
+import React from "react";
+import { Doughnut } from "react-chartjs-2";
+import styles from "../styles/CircleChart.module.scss";
+
+export default function CircleChart({ data }) {
+  const chartLabels = data.map((p) => Object.keys(p));
+  const chartData = data.map((p) => Object.values(p));
+  const config = {
+    labels: [...chartLabels],
+    datasets: [
+      {
+        data: [...chartData],
+        backgroundColor: [
+          "#FF6384",
+          "#36A2EB",
```

Comment:
We can put these into a different file so that all of us can access the same colors across different charts